src/js/webrequest.js

```diff
@@ -713,6 +713,19 @@ function dispatcher(request, sender, sendResponse) {
       tabUrl: tab_url
     });
 
+  } else if (request.type == "getOptionsData") {
+    let settings = badger.storage.getBadgerStorageObject("settings");
```

Comment:
The storage object is called `"settings_map"`.

Reply:
Done